share/instruments/instruments.xml

```diff
@@ -3246,6 +3246,7 @@
                   </Channel>
             </Instrument>
             <Instrument id="baritone-horn">
+                  <trackName>Baritone Horn</trackName>
```

Comment:
Why this addition?

Reply:
Sharp eye ;-)
This is to be able to translate this better into German (as "Tenorhorn (Mitteleuropa)"), independantly from the long instrument name ("Tenorhorn"), as that's what this instrument is called in German (and Czech and maybe a few more)

Reply:
Baritone Horn is translated as Tenorhorn in German? :P

Reply:
Yes, And "Alto Horn" to "Tenor Horn" in the UK, just to add to the confusion

Reply:
I do not feel that this is necessary as there is no such distinction in English and it becomes really confusing for the user.

Reply:
If you read the issue and the forum topic that led to is, you should see that it is needed, and exactly to avoid confusion.

Reply:
But feel free to discuss it there

Reply:
Anyway, as that is something only the German translation would ever use to translate it to something different (see above), there's no harm to anyone else.
The long and short instrument names are already translated that way, but without that "(Central Europe)".

Reply:
It seems to be an unnecessary addition to the software that would be used by a very small number of people while adding for ALL users yet another instrument in a very, very long list.

I would be impressed if more than 10 users would use this instrument over the next year.

A better way to approach this could be to have a standard instrument list in the default installation, then add an complete instrument list through an extension.

Reply:
This line does not add any instrument

Reply:
If you want to discuss the necessity of the **added** instrument, you're at the wrong spot.
And in the (German) forum discussion 3 people already voiced their need of that addition.

Reply:
I would guess not more than 10 users will ever write for about the half the instruments on the list (eg, Greatbass Crumhorn).  But that's why we have the genre selectors and in particular the "Common" instruments listed by default - to keep the less common ones from cluttering the display.  And also why we have the search facility.  I don't see this particular instrument as any less appropriate to include than any of a hundred others.

Reply:
Thanks, and indeed those are not on the Common list

Reply:
I am a bit confused here...

There is an existing instrument that is a Tenorhorn (in Eb, called Althorn in German).
There is an existing instrument that is a Baritone Horn (in Bb, called Tenorhorn in German).
These is an existing instrument that is a Euphonium (in Bb, called Euphonium in German).

These are essentially the same interments. The differences between them are small, like a preference among DACH manufacturers (and Chinese clones) to produce rotary valves vs. piston and the Baritone Horn (Tenorhorn in German) that these manufacturers produce has a different bore which produces a slightly different sound and it has a more limited range.

Anyway, I don't understand the need to change the name in English due to the German translation or the need to add a new instrument as the transpositions are not different, only slight difference in range.

Reply:
Then read again. That Alto Horn is Tenor Horn in the UK (and there only)
The Bariton Horn is a Tenor Horn in Germany (and some neighbors), and the added Bariton Horn is not the same, same sound, but different range, similar but not identical as Euphonium

Reply:
Alto Horn is Tenor Horn in all English speaking countries. 

So, you are saying that this is to add a new instrument?

There is such a really small distinction on this instrument for such a really small audience that I do not believe that it is deserving of a distinct instrument. Is there any software in the entire world that supports this as a distinct instrument?

Do we really need to support every single instrument when there is an alternative that is 90% similar? It makes sense to add micro detail into an extension, but not fill the core software with unnecessary stuff.

Reply:
It is in the UK, as far as I know
It an extension it won't get translated

Reply:
Isn't the problem 90% solved by changing the English world Tenor Horn to Althorn in German and Baritone Horn to Tenorhorn in German?

Yes, the range is slightly different, but is it really different enough to deserve an entirely new instrument? That is my whole point. 

This seems like the difference between a French Bassoon and German or French Clarinet and German. 

They are different instruments, with slightly different ranges and absolutely different fingerings, but there is not enough of a distinction to make them deserving of distinct instruments... despite the fact there are greater differences than between the Baritone Horn and the German Tenorhorn.

My argument is simply for clarity for the user. While this will make it much clearer for 1% of users, it will confuse 99% who have never heard of such a thing.

Reply:
There is no Tenor Horn. Just an Alto Horn, which for that UK is translated to Tenor Horn.
And for the German Tenor Horn it has an entire wrong range, but the English Bariton Horn has the right range and sound for the German Tenor Horn, so is not translated as such. But then there is a German Bariton Horn missing. It has the same sound as the other Horns (French Horn), but a range similar (not identical) to Euphonium, but that in turn has a Tuba sound.

Reply:
I don't think the necessity should get discussed here, but in the issue and the forum topic that triggered the issue,  as only there the people requesting is would see and participate in the discussion. Here we can discuss the details of the implementation.